main.c

```diff
@@ -132,6 +137,20 @@ static void keyboard_handle_key(void *data, struct wl_keyboard *wl_keyboard,
 		if (key == KEY_ESC) {
 			state->running = false;
 		}
+
+		if (key == KEY_SPACE) {
```

Comment:
Note: Weird behavior if holding `space` before clicking.
The selection starts at the top left corner to the mouse position. Any mouse movement after that works like intended with or without releasing `space`.

Should we check for the selection before setting `edit_anchor`?

Reply:
We could check `button_state` to prevent this?

Reply:
Yeah make sense, and ignore the fact that the user want to move the selection since there is no selection to move around. If the user wants to move it, he would have to release and press again.

Seems good :+1:

Reply:
Checked `seat->has_selection` instead. Felt cleaner.